### PR TITLE
IDEM-9

### PR DESCRIPTION
**Pull request by bitcoinbrisbane**
- Refactor PGP to own utils
- Add unit tests

### CODE REVIEW

**Review by bitcoinbrisbane — COMMENTED** [review]
LGTM

File: .expo/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+- "devices.json": contains information about devices that have recently opened this project. This is used to populate the "Development sessions" list in your development builds.
+- "packager-info.json": contains port numbers and process PIDs that are used to serve the application to the mobile device/simulator.
+- "settings.json": contains the server configuration that is used to serve the application manifest.
+
+> Should I commit the ".expo" folder?
+
+No, you should not share the ".expo" folder. It does not contain any information that is relevant for other developers working on the project, it is specific to your machine.
+
+Upon project creation, the ".expo" folder is already added to your ".gitignore" file.
```
Comment (amagon270) [inline .expo/README.md:15]: why has the .expo folder been uploaded?  shouldn't it be ignored in the .gitignore?

File: app/src/hooks/usePpg.ts [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    publicKey: string | undefined,
+    email: string | undefined
+  ) => Promise<void>;
+  pgp: PGP | undefined;
+  setPGP: (pgp: PGP) => void;
+  verifyPGPPublicKey: (email: string | undefined) => Promise<void>;
+};
+
+const usePgp = (): Hooks => {
+  const [pgp, setPGP] = React.useState<PGP | undefined>(undefined);
```
Comment (amagon270) [inline app/src/hooks/usePpg.ts:26]: As far as I can tell this variable isn't used anywhere.  Is there a reason we have it?

**Review by amagon270 — COMMENTED** [review]
Few comments

**Review by amagon270 — APPROVED** [review]

**Review by pondpiu — APPROVED** [review]
tick

**Pull request merged**